01_values.md

```diff
@@ -1,359 +1,229 @@
 {{meta {docid: values}}}
 
-# Values, Types, and Operators
+# Valores, Tipos, e Operadores
 
 {{quote {author: "Master Yuan-Ma", title: "The Book of Programming", chapter: true}
 
-Below the surface of the machine, the program moves. Without effort,
-it expands and contracts. In great harmony, electrons scatter and
-regroup. The forms on the monitor are but ripples on the water. The
-essence stays invisibly below.
+> Sob a superfície da máquina, o programa se move. Sem esforço, se expande e se contrai. Em grande harmonia, elétrons se espalham e se reagrupam. As formas no monitor são nada mais que ondulações na água. A essência permanece invisível abaixo.
+>
+> — Master Yuan-Ma, The Book of Programming
 
 quote}}
 
 {{index "Yuan-Ma", "Book of Programming"}}
 
-{{figure {url: "img/chapter_picture_1.jpg", alt: "A sea of bits", chapter: framed}}}
+{{figure {url: "img/chapter_picture_1.jpg", alt: "Mar de bits", chapter: framed}}}
 
 {{index "binary data", data, bit, memory}}
 
-Inside the computer's world, there is only data. You can read data,
-modify data, create new data—but that which isn't data cannot be
-mentioned. All this data is stored as long sequences of bits and is
-thus fundamentally alike.
+Dentro do mundo do computador, só existem dados. Você pode ler dados, modificar dados, criar novos dados — mas aquilo que não é dado não pode ser mencionado. Todos esses dados são armazenados como longas sequências de bits e são assim fundamentalmente iguais.
 
 {{index CD, signal}}
 
-_Bits_ are any kind of two-valued things, usually described as zeros and
-ones. Inside the computer, they take forms such as a high or low
-electrical charge, a strong or weak signal, or a shiny or dull spot on
-the surface of a CD. Any piece of discrete information can be reduced
-to a sequence of zeros and ones and thus represented in bits.
+_Bits_ são quaisquer tipos de coisas com dois valores possíveis, geralmente descritos como zeros e uns. Dentro do computador, eles tomam formas como uma carga elétrica alta ou baixa, um sinal forte ou fraco, ou um ponto na superfície de um CD que tem brilho ou não. Qualquer pedaço de informação pode ser reduzido a uma sequência de zeros e unse, portanto, representado em bits.
 
 {{index "binary number", radix, "decimal number"}}
 
-For example, we can express the number 13 in bits. It works the same
-way as a decimal number, but instead of 10 different ((digit))s, you
-have only 2, and the weight of each increases by a factor of 2 from
-right to left. Here are the bits that make up the number 13, with the
-weights of the digits shown below them:
+Por exemplo, podemos expressar o número 13 em bits. Funciona do mesmo jeito como uma representação decimal mas ao invés de 10 dígitos diferentes, você tem apenas dois e o peso de cada um aumenta por um fator de 2, da direita para a esquerda. Estes são os bits que formam o número 13, com o peso de cada um logo abaixo:
 
-```{lang: null}
+```
```

Comment:
Você poderia manter `{lang: null}`, como o original aqui?

_ps. vamos fazer assim no primeiro round e, se for o caso, fazemos essas alterações no futuro.
o foco inicial é apenas traduzir a as partes que contêm textos traduzíveis._